manifests/redirect.pp

```diff
@@ -58,6 +58,7 @@
     $ssl_certificate = undef,
     $ssl_certificate_key = undef,
     $ssl_dh_param = undef,
+    $extra_headers = {},
```

Comment:
No, because this way you can do something like this:

```
nginx::redirect {'redirect-ssl-api.irail.be':
    server_name => 'api.irail.be',
    target      => 'https://api.irail.be',
    enable_ipv6 => true,
    extra_headers => {
       'Access-Control-Allow-Origin' => '*',
       'Access-Control-Allow-Method' => 'GET,POST',
    },
  }
```